{"name": "test_fun1", "status": "failed", "statusDetails": {"message": "AssertionError: assert 10 != 10", "trace": "def test_fun1():\n        print(\"test 2\")\n>       assert 10!=10\nE       assert 10 != 10\n\ncomponents/adt/tests/test_case.py:7: AssertionError"}, "attachments": [{"name": "stdout", "source": "fabc2576-3d95-4172-a896-b230fcde6869-attachment.txt", "type": "text/plain"}], "start": 1632123996058, "stop": 1632123996059, "uuid": "a55b6163-93b8-41ba-a51b-2448fbbad16a", "historyId": "f96aa4659f172d70357ba5e793f922b1", "testCaseId": "770873b83a9f56338e8b6ed17475b4da", "fullName": "components.adt.tests.test_case#test_fun1", "labels": [{"name": "parentSuite", "value": "components.adt.tests"}, {"name": "suite", "value": "test_case"}, {"name": "host", "value": "Muhammads-MacBook-Pro.local"}, {"name": "thread", "value": "9567-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "components.adt.tests.test_case"}]}